src/components/card/Description.vue

```diff
@@ -262,6 +262,8 @@ export default {
 
 #description-preview {
 	min-height: 100px;
+	width: max-content;
+	overflow-x: scroll;
```

Comment:
```suggestion
	overflow-x: auto;
```

Let's drop the max-content as that would also avoid long lines to break - and the auto seems to work fine to not always have a scrollbar in some browsers.